src/tennisSimulator/Database.java

```diff
@@ -0,0 +1,40 @@
+package tennisSimulator;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class Database {
```

Comment:
I think you should be more specific with naming.
I terms of Tennis game, Database can be anything. Is it Game database? Or is it Player database? Maybe even something else.

Reply:
Changed into `GameDatabase`. Possibly in future this class may contain list of available game modes.